CHANGELOG.md

```diff
@@ -41,6 +41,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Notes
 - Version was intended for PyPI release but encountered deployment issues.
 
+### Documentation
+- Added new pages to Sphinx documentation:
+    - CCVM Simulator - Equations of Motion
+    - Stochastic Differential Equations (SDE) for Langevin dynamics
+    - Stochastic Differential Equations (SDE) for the DL-CCVM
+    - Stochastic Differential Equations (SDE) for the Measurement-Feedback Coherent Continuous Variable Machine (MF-CCVM)
```

Comment:
```suggestion
    - Stochastic Differential Equations (SDE) for the Measurement-Feedback Coherent Continuous-Variable Machine (MF-CCVM)
```

Reply:
Additional comments for `CHANGELOG.md` in order to avoid redundant explanation:
I suggest we should keep line 47 as it is and then change the rest as follows: 
48:  - SDE for the DL-CCVM
49:  - SDE for the MF-CCVM

Reply:
Thanks Mehmet!
It looks like these were meant to be the exact title of each page that was added, but now that you point it out, the naming choices for the DL-CCVM and MF-CCVM pages are inconsistent. I'll fix that, too.

Reply:
@KatieO4t1qbit, in line 47, why isn't it "the Langevin dynamics", as you have it elsewhere?
Also, if you're using the expanded form on line 49, you should do so on line 48, but you should also capitalize "Dynamics" on line 47.

Reply:
I'm going to leave this comment until we've discussed my idea above, which involves removing "the" from all the titles.

Reply:
@KatieO4t1qbit 
I'm not sure removing "the" from the titles fits with the meaning we have in the research paper. In the paper, we use "the" when referring to something that is a machine or a solver: "the ... machine". 
Moreover, writing "_the_ Langevin Dynamics", where we could take "solver" as being implied but it would be probably be better to include the word "Solver", helps show we are not speaking about Langevin dynamics in general.

Perhaps we could discuss this before it's been finalized.

Reply:
Great, thanks Marko! I'll update that.